qemu/tests/netperf_stress.py

```diff
@@ -39,7 +39,10 @@ def run(test, params, env):
     status_test_command = params.get("status_test_command", "echo $?")
     compile_option_client = params.get("compile_option_client", "")
     compile_option_server = params.get("compile_option_server", "")
-    host_ip = utils_net.get_host_ip_address(params)
+    if params.get("netperf_vlan_test") and params.get("host_vlan_ip"):
```

Comment:
What is the value of `params.get("netperf_vlan_test")`? 
You may want to leave a comment in the cfg.
Unless the param `netperf_vlan_test` is not provided, otherwise `if params.get("netperf_vlan_test")` will always be `True` .
Same goes to `params.get("host_vlan_ip")`.

Reply:
@hereischen Thanks for review, I want to provide `netperf_vlan_test` and `host_vlan_ip` in bridge_vlan.cfg and other vlan case script, it will not be provided in other netperf cases.